buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidTarget.groovy

```diff
@@ -46,12 +47,33 @@ abstract class AndroidTarget extends JavaLibTarget {
 
         // Butterknife support
         generateR2 = project.plugins.hasPlugin('com.jakewharton.butterknife')
+
+        // SqlDelight support
+        if (project.plugins.hasPlugin('com.squareup.sqldelight')) {
+            File sqldelightDir = new File(project.projectDir, "build/okbuck/sqldelight")
+            if (!sqldelightDir.exists()) {
```

Comment:
i mean you can directly call `mkdirs()`

if it exists, it will continue quietly. if it does not, it will create it

no need for the if block